share/spice/mars_photos/mars_photos.js

```diff
@@ -0,0 +1,33 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_mars_photos = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('mars_photos');
+        }
+
+        Spice.add({
+            id: "mars_photos",
+            name: "Mars Photos",
+            data: api_result.photos,
+            meta: {
+                sourceName: "nasa.com",
```

Comment:
Nitpick: We don't usually put the `.com` part like "Amazon" in https://duckduckgo.com/?q=ray+bradbury+books&ia=products.

So this should probably be "NASA".